cost-analyzer/values.yaml

```diff
@@ -3108,7 +3108,8 @@ costEventsAudit:
   enabled: false
 
 ## Disable updates to kubecost from the frontend UI and via POST request
-##
+## This feature is considered beta, entrprise users should use teams:
```

Comment:
Not sure it's right to call a long-standing feature "beta"? Maybe comment should be changed to:

> Enterprise user are recommended to use the Teams feature:

Reply:
Good point. but I think it can be promoted when the UI 1. tells the user it is read only. 2. UI doesn't throw errors when trying to write settings.